src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1682,6 +1682,15 @@ public int getEnd() {
         return (int)stop;
     }
 
+    /**
+     *
+     * @return whether the variant context meets the requirements to be a REF-only block
+     *
+     */
+    public boolean isRefOnlyBlock(){
+        return getAlternateAlleles().size() == 1 && getAlternateAllele(0).isSymbolic() && getAttribute(VCFConstants.END_KEY) != null;
```

Comment:
hmmm. shouldn't you check that the allele is `<NON_REF>` or `<*>`?

Reply:
I wasn't sure how to interpret the spec.  Whether it meant any symbolic allele could be used to represent the unspecified alternate allele or it had to specifically be "<*>".  Should I restrict it to just those two strings (one from the spec and one from gvcf land)?